docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -65,6 +65,24 @@ def downstream_asset(upstream_asset):
 
 The [explicit dependencies](#explicit-dependencies) example covers an alternative way to specify asset dependencies without needing to match argument names to upstream asset names.
 
+### Graph-backed assets
+
+[Basic software-defined assets](#a-basic-software-defined-asset) can only produce one data artifact. If your use case requires an asset that produces more than one data artifact, you can use graph-backed assets.
+
+This approach to asset creation allows you to define a graph of ops which can produce one or multiple assets. **Note**: To use graphs to create an asset, the graph **must return an output**.
+
+To define a graph-backed asset, use the `from_graph` attribute on the `AssetsDefinition` object:
+
+<TODO />
+
+If creating a job using a graph-backed asset, select all graph outputs using `asset_selection`:
+
+<TODO />
+
+**Note**: All graph outputs must be selected when using a graph-backed asset to create a job.
```

Comment:
Let's make this note more specific by saying:
All output assets must be selected when using a graph-backed asset to create a job.